worker/worker/worker.js

```diff
@@ -247,11 +250,20 @@ async function handler_instagram_login(request) {
   }
 }
 
+async function handler_data_deletion(request) {
+  return new Response(JSON.stringify({
+    code: `${+ new Date.now()}`,
```

Comment:
What does this do?

Reply:
It it supposed to provide a confirmation that the authorization data is deleted, but we don't store any. This conforms to the FB requirements, as far as I can tell. Most importantly, I direct the end user to a page that states we don't store any authorization data.